dockers/docker-fpm-frr/frr/supervisord/supervisord.conf.j2

```diff
@@ -141,6 +141,20 @@ stderr_logfile=syslog
 dependent_startup=true
 dependent_startup_wait_for=bgpd:running
 
+{% if DEVICE_METADATA.localhost.frr_mgmt_framework_config is defined and DEVICE_METADATA.localhost.frr_mgmt_framework_config == "true" %}
```

Comment:
why we have this if condition ? seems completely unrelated

Reply:
see the line# 128, because staticroutebfd need to work together with bgpcfgd, so use same condition with bgpcfgd